ansible/roles/hypothesis/templates/initialize.d/hypothesis.sh

```diff
@@ -17,9 +17,12 @@ function wait_url {
 }
 
 if [ ! -e libexec/secrets.sh ]; then
+  set +x
+  echo "Creating secret keys..."
   echo "export SECRET_KEY='$(openssl rand -base64 32 | tr -d '\n')'" >> libexec/secrets.sh
```

Comment:
So if the keyfile already exists, we append a new key to it? :)

Reply:
See the `~` before the `-e`?

Reply:
Now that you mention it, yes, I do :) But then, why `>>`? It's confusing.

Reply:
Yeah, `>` looks better.